src/data_loader.py

```diff
@@ -77,30 +77,31 @@ def load_data(self, file_path_list):
                     current_date_time = current_datetime()
                     id = self.get_record_id(intention, exist_node)
                     crdc_id = self.get_crdc_id(intention, exist_node, type, node_id)
-                    dataRecord = {
-                        ID: id,
-                        CRDC_ID: crdc_id if crdc_id else id,
-                        SUBMISSION_ID: self.batch[SUBMISSION_ID],
-                        DATA_COMMON_NAME: self.data_common,
-                        BATCH_IDS: batchIds,
-                        "latestBatchID": self.batch[ID],
-                        "uploadedDate": current_date_time, 
-                        STATUS: STATUS_NEW,
-                        ERRORS: [],
-                        WARNINGS: [],
-                        CREATED_AT : current_date_time if intention == INTENTION_NEW or not exist_node else exist_node[CREATED_AT], 
-                        UPDATED_AT: current_date_time, 
-                        "orginalFileName": file_name,
-                        "lineNumber":  index + 2,
-                        "nodeType": type,
-                        "nodeID": node_id,
-                        PROPERTIES: {k: v for (k, v) in rawData.items() if k in prop_names},
-                        PARENTS: self.get_parents(relation_fields, row),
-                        "rawData":  rawData
-                    }
-                    if type in file_types:
-                        dataRecord[S3_FILE_INFO] = self.get_file_info(type, prop_names, row)
-                    records.append(dataRecord)
+                    if index == 0 or not self.has_m2m_rel(records, node_id, rawData, relation_fields):
```

Comment:
To check many to many relationship of current row with existing node in records, the first row is skipped since there are not dataRecords converted from a row and inserted int records list when index is 0.